specification.md

```diff
@@ -1678,9 +1706,14 @@ The message has the following structure:
 ```protobuf
 message AgentRemoteConfig {
   AgentConfigMap config = 1;
+  string new_instance_uid = 2;
```

Comment:
I wasn't sure about the right place for this field. Did not want to create a new message and concluded that in fact it's part of the config, but I am open to suggestions on moving it elsewhere. Since we want to support multiplexing when setting instance_uid, it needs to be separated from the "ServerToAgent.instance_uid"

Reply:
I do not think it fits well in the AgentRemoteConfig. The "config" is defined to mean a set of opaque byte sequences to the protocol. This now violates that notion.
A new message sounds better to me.

Reply:
Sure, added `AgentIdentification` as a new field